security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-operator-issuer-acme"]
+= Managing certificates with an ACME issuer
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-issuer-acme
+
+toc::[]
+
+TODO
+
+// TODO: Is this the same as Let's Encrypt? need to mention that at all?
```

Comment:
@thejasn @swghosh What is the relationship between ACME and Let's Encrypt? Is Let's Encrypt a vendor or implementation of ACME issuer? 

Basically, do we need to mention Let's Encrypt at all in our docs, or is using ACME throughout fine?

Reply:
Let's encrypt is a certificate authority that uses ACME protocol. So the two aren't really interchangeable.

Reply:
Letsencrypt is just one of the public CAs that support acme protocol and there are other such CAs as well (eg: https://zerossl.com).